### PR TITLE
softnewlines option

### PR DESCRIPTION
**Pull request by Nihaux**
ability to disable the behaviour of inserting soft new line when user hit shift + enter

### CODE REVIEW

File: src/components/MegadraftEditor.js [hunk 1]
```diff
@@ -24,9 +24,11 @@ import DEFAULT_ENTITY_INPUTS from "../entity_inputs/default";
 export default class MegadraftEditor extends Component {
   constructor(props) {
     super(props);
-
     this.state = {
-      readOnly: this.props.readOnly || false
+      readOnly: this.props.readOnly || false,
+      softNewLines: this.props.softNewLines !== undefined
```
Comment (vierno) [inline src/components/MegadraftEditor.js]: Hello @Nihaux , thanks for contributing with Megadraft.
I believe you don't need to change the component's state, could you change the code by checking if the prop === false on the handleReturn method?

Comment (Nihaux) [inline src/components/MegadraftEditor.js]: yep, ill change that

File: tests/components/MegadraftEditor_test.js [hunk 2]
```diff
@@ -202,6 +202,18 @@ describe("MegadraftEditor Component", () => {
...
+  it("does not insert soft line breaks if option set to false", function () {
+    const wrapper = mount(
+      <MegadraftEditor
+        editorState={this.editorState}
+        onChange={this.onChange}
+        softNewLines={false} />
+    );
+    const component = wrapper.get(0);
+    expect(component.handleReturn({shiftKey: true})).to.be.equal(false);
+    expect(this.onChange.called).to.be.equal(false);
```
Comment (marcelometal) [inline tests/components/MegadraftEditor_test.js]: I believe you don't need this line (214) anymore

**Comment by Nihaux** [comment]
@marcelometal done, sry for the delay

**Comment by marcelometal** [comment]
Thank you @Nihaux !

**Comment by ricobl** [comment]
Thanks @Nihaux! It's released under the `0.4.12` version.

**Pull request merged**